ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/AddExpenseClaimActivity.java

```diff
@@ -0,0 +1,122 @@
+// Source for DatePicker: http://androidopentutorials.com/android-datepickerdialog-on-edittext-click-event/
+
+package ca.ualberta.cs.shinyexpensetracker;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaim;
+import android.app.DatePickerDialog.OnDateSetListener;
+import android.app.Activity;
+import android.app.DatePickerDialog;
+import android.app.Dialog;
+import android.os.Bundle;
+import android.text.InputType;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.widget.DatePicker;
+import android.widget.EditText;
+
+public class AddExpenseClaimActivity extends Activity implements OnClickListener {
```

Comment:
A Javadoc comment at the top of every class describing what it is, what it does, etc.